### ISSUE TITLE
General Management Page: Ability to change SSID & password remotely

### ISSUE THREAD

**Issue by guruthree** [Blayzeing/robo-pad#3]
This could probably be stored in EEPROM or SPIFFs (https://tttapa.github.io/ESP8266/Chap11%20-%20SPIFFS.html) directly.

**Comment by Blayzeing** [comment]
Yeah, the SPIFF is something I want to look at more, I just think it's
easier with the IDFvtoolchain than Arduino.

**Comment by guruthree** [comment]
For just a little bit of data, EEPROM should be fine, looks pretty easy to use: https://arduino-esp8266.readthedocs.io/en/latest/libraries.html#eeprom

**Comment by Blayzeing** [comment]
Ah yes, of course - that makes a lot more sense haha - I was mixing it up with my efforts to store the page on the SPIFFs without making burning more complicated for new users.

But having a password (or, indeed, general) management page on-device would be good.
